livekit-rtc/livekit/rtc/room.py

```diff
@@ -436,6 +439,28 @@ async def _listen_task(self) -> None:
             self._room_queue.put_nowait(event)
             await self._room_queue.join()
 
+    def _on_rpc_method_invocation(self, rpc_invocation: RpcMethodInvocationEvent):
+        if self._local_participant is None:
+            logging.warning(
+                "Received RPC invocation before local participant was initialized"
+            )
+            return
+
+        if (
+            rpc_invocation.local_participant_handle
+            == self._local_participant._ffi_handle.handle
+        ):
+            asyncio.create_task(
```

Comment:
We need to keep a reference to the tasks. (See https://docs.python.org/3.12/library/asyncio-task.html#asyncio.create_task)
It's also unclear when we should cancel/terminate them (at room shutdown?)

Can you also explicitly use the loop of the room (self._loop)

Reply:
Ok so I did this and went ahead and cancelled them on room disconnect. I also added tests for this behavior and timeouts in the example and found a couple small issues that I addressed. I also decided to switch response_timeout_ms to response_timeout in float seconds to be more pythonic and I'll make the a similar change in node and JS to drop the `ms` part of the name and be more platform-specific.